src/blocks/form/edit.js

```diff
@@ -289,6 +290,7 @@ class FormEdit extends Component {
 				</Notice>
 				<TextControl
 					label={ __( 'Subject', 'coblocks' ) }
+					ref={ this.subjectField }
```

Comment:
What is the purpose for using a ref here? Can we instead use `this.state` within the component?

Reply:
Good idea. Made those changes.